sdk/storage/storage-blob-changefeed/src/BlobChangeFeedClient.ts

```diff
@@ -0,0 +1,119 @@
+import { BlobServiceClient } from "@azure/storage-blob";
+import { PagedAsyncIterableIterator, PageSettings } from "@azure/core-paging";
+import { BlobChangeFeedEvent } from "./models/BlobChangeFeedEvent";
+import { ChangeFeedFactory } from "./ChangeFeedFactory";
+import { ChangeFeed } from "./ChangeFeed";
+import { CHANGE_FEED_MAX_PAGE_SIZE } from "./utils/constants";
+
+export interface BlobChangeFeedGetChangesOptions {
+  start?: Date;
+  end?: Date;
+}
+
+export class BlobChangeFeedEventPage {
+  public events: BlobChangeFeedEvent[];
+  public continuationToken: string;
+
+  constructor() {
+    this.events = [];
+    this.continuationToken = "";
+  }
+}
+
+export class BlobChangeFeedClient {
+  /**
+   * blobServiceClient provided by @azure/storage-blob package.
+   *
+   * @private
+   * @type {BlobServiceClient}
+   * @memberof BlobChangeFeedClient
+   */
+  private _blobServiceClient: BlobServiceClient;
+  private _changeFeedFactory: ChangeFeedFactory;
+
+  public constructor(blobServiceClient: BlobServiceClient) {
```

Comment:
Do we want to have a custom user agent prefix for change feed? If we were passing in pipeline options, we probably could add something before passing the options to BlobServiceClient constructor.

Reply:
Good idea. What other customer options for change feed that you can think of?
@XiaoningLiu @jiacfan

Reply:
@ljian3377 What does other languages do? We can pick up one like blob/queue, and later you can update storage backend useragent analysis job to support this new agent format.

Reply:
.NET doesn't have it for now. But does userAgent specially apply for JS?

Reply:
Probably worth to discuss, but this should be cross languages.

Reply:
And not blocking preview.